bin/kn

```diff
@@ -45,6 +48,26 @@ if [ -z "$KUBENOW_VERSION" ] ; then
   KUBENOW_VERSION="latest"
 fi
 
+# Set directories to be mounted in docker
+USER_DIR="$PWD"
+USER_DIR_DOCKER="/var/userdir"
+# This is mainly for development where a local kubenow-dir will
+# be mounted on top of the kubenow dir in docker image
+if [ -n "$KUBENOW_DIR" ] ; then
+  KUBENOW_DIR_DOCKER="/opt/KubeNow"
+  OPTIONAL_KUBENOW_DIR_MNT="-v $KUBENOW_DIR:$KUBENOW_DIR_DOCKER"
```

Comment:
No need for an extra var, please add this straight in the `docker run`, it is simpler to understand what is going on in that way.

Reply:
I don't see how I could add it in docker run. The whole mount line including "-v" is optional, not just the name of directory. Thats why it is not cited below (so -v is read as argument).

Reply:
This var will also disappear soon if we copy the script, playbooks and bin to user_dir